src/DependencyInjection/Compiler/DriverPass.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace EzSystems\PlatformHttpCacheBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+class DriverPass implements CompilerPassInterface
```

Comment:
Nitpick: Please add brief description on what this Compiler Pass does.

Reply:
fixed